core/blockchain.go

```diff
@@ -1741,7 +1741,7 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals, setHead bool)
 				throwaway, _ := state.New(parent.Root, bc.stateCache, bc.snaps)
 
 				go func(start time.Time, followup *types.Block, throwaway *state.StateDB, interrupt *uint32) {
-					bc.prefetcher.Prefetch(followup, throwaway, bc.vmConfig, &followupInterrupt)
+					bc.prefetcher.Prefetch(followup, throwaway, bc.vmConfig, interrupt)
```

Comment:
Ah, good catch, but I think this was probably some leftover from a previous code or refactor. There's no need really for this to be passed as a parameter to the function. Instead of your change, we should IMO just drop the `interrupt *uint32` param.

Reply:
I mean drop if from this anonymous goroutine func.

Reply:
updated